packages/angular/package.json

```diff
@@ -43,10 +44,12 @@
     "tslib": "^2.3.1"
   },
   "peerDependencies": {
+    "@unovis/shared": "*",
+    "@unovis/ts": "^1.2.1",
```

Comment:
@ivegotwings I think we need to specify the exact version here because we can't really guarantee the support of newer versions.